app/routes/api-router.js

```diff
@@ -4,11 +4,11 @@ var router = express.Router();
 var loginAPI = require("./login-api");
 var userAPI = require("./user-api");
 var meetingAPI = require("./meeting-api");
-var senmailAPI=require("./sendmail-api");
+var senmailAPI = require("./sendmail-api");
```

Comment:
We don't really need a sendmailAPI, the api I talked about is used to check peter's work in investigation phase. A module is enough. And an api may cause risk.